api/src/main/resources/messages.properties

```diff
@@ -9,4 +9,4 @@ legacyui.manageuser.noProviderIdentifier=No Identifier Specified
 
 ${project.parent.artifactId}.Location.purgeLocation=Permanently Delete Location
 ${project.parent.artifactId}.Location.confirmDelete=Are you sure you want to delete this Location? It will be permanently removed from the system.
-${project.parent.artifactId}.Location.purgedSuccessfully=Location deleted successfully
```

Comment:
Do we need this change?

Reply:
that was un intended , i cant even figure out the change (: